src/openrct2-ui/windows/EditorObjectiveOptions.cpp

```diff
@@ -952,6 +952,7 @@ static void window_editor_objective_options_main_paint(rct_window* w, rct_drawpi
     // Park name
     x = w->windowPos.x + 8;
     y = w->windowPos.y + w->widgets[WIDX_PARK_NAME].top;
+    ScreenCoordsXY coords = { x , y };
```

Comment:
Either construct them in place on the calls { x, y } or replace `x, y` in the whole function by `screenCoords` otherwise this is bound to be forgotten to be updated.